app/controllers/jobseekers/job_applications_controller.rb

```diff
@@ -110,6 +110,12 @@ def post_submit
 
   def show
     raise ActionController::RoutingError, "Cannot view draft application" if job_application.draft?
+
+    if params[:tab] == "messages"
+      conversation = job_application.conversations.first
+      @messages = conversation&.messages&.order(created_at: :desc) || []
```

Comment:
could we not just display all messages here and so avoid this safe-nav operators? (which look scary IMHO)